### PR TITLE
[CRS-281] MML integration

### PR DESCRIPTION
**Pull request by gumuz**
Integrating the mml lib into, will not work correctly without these: https://github.com/GetStream/mml-react/pull/1

### CODE REVIEW

File: src/components/Message/hooks/useActionHandlerMML.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// @ts-check
```
Comment (mahboubii) [inline src/components/Message/hooks/useActionHandlerMML.js:1]: this is basically the same as `useActionHandler`
https://github.com/GetStream/stream-chat-react/pull/562/files#diff-aa50df8b4b857ab266247a60ceb2e55f

File: src/components/Message/MessageText.js [hunk 2]
```diff
@@ -58,6 +61,12 @@ const MessageTextComponent = (props) => {
     return null;
   }
 
+  const renderedText = message?.mml ? (
+    <MML source={message.mml} onSubmit={handleActionMML} />
+  ) : (
+    messageText
+  );
```
Comment (mahboubii) [inline src/components/Message/MessageText.js:68]: IMO, we should stick to the attachment type, mixing up the text render with the mml could make things unnecessary complicated

**Review by mahboubii — CHANGES_REQUESTED** [review]
@gumuz are we going to support mml in `message.mml` and `message.attachments[{type: mml}]` ?

**Comment by gumuz** [comment]
> @gumuz are we going to support mml in `message.mml` and `message.attachments[{type: mml}]` ?

Honestly, I'm not sure. I guess you can already use attachment types in any way you want at the moment, does not need any backend support afaik. Thoughts @tbarbugli ?

ps. this PR was a proof of concept, feel free to use/discard it however you want.

**Pull request closed without merging**